pyart/graph/radardisplay.py

```diff
@@ -1041,6 +1180,35 @@ def _get_ray_data(self, field, ray, mask_tuple, filter_transitions):
 
         return data
 
+    def _get_azimuth_rhi_data_x_y_z(self, field, target_azimuth,
+                               edges,mask_tuple):
+        """Retrieve and return pseudo-RHI data from a plot function. """
+        # determine which rays from the ppi radar make up the pseudo RHI
+        data = self.fields[field]['data']
+
+        if mask_tuple is not None:
+            mask_field, mask_value = mask_tuple
+            mdata = self.fields[mask_field]['data'][ray]
```

Comment:
ray is not defined in this scope and I don't believe a slice is required here at all.